documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -18,23 +19,56 @@ Client secret:: A client secret contains the _clients CA_ for a user to sign its
 This allows mutual authentication against the Kafka cluster. The broker validates a client's identity through the certificate.
 User secret:: A user secret contains a private key and certificate. The secret is created and signed by the clients CA when a new user is created. The key and certificate are used to authenticate and authorize the user when accessing the cluster.
 
-== Secrets in PEM and PKCS #12 formats
+[id='certificates-and-secrets-formats-{context}']
+== TLS authentication using keys and certificates in PEM or PKCS #12 format
 
-Secrets provide private keys and certificates in PEM and PKCS #12 formats.
-Use the format that's suitable for your client.
-Using private keys and certificates in PEM format means that users have to get them from the secrets,
-and generate a corresponding truststore or keystore to use in their applications.
-PKCS #12 storage provides a truststore or keystore that can be used directly.
+The secrets created by Strimzi provide private keys and certificates in PEM (Privacy Enhanced Mail) and PKCS #12 (Public-Key Cryptography Standards) formats.
+PEM and PKCS #12 are OpenSSL-generated key formats for TLS communications using the SSL protocol.
 
-PKCS #12 defines a single file store (`.p12`) for cryptography objects with password protection.
-You can use PKCS #12 to manage certificates and keys in one place.
+You can configure mutual TLS (mTLS) authentication that uses the credentials contained in the secrets generated for a Kafka cluster and user. 
 
-Each secret contains properties specific to PKCS #12.
+To set up mTLS, you must first do the following:
 
-* The `.p12` property specifies the store for certificates and keys.
-* The `.password` property specifies the password that protects the store.
+* xref:con-securing-kafka-authentication-{context}[Configure your Kafka cluster with a listener that uses mTLS]
+* xref:con-securing-client-authentication-{context}[Create a `KafkaUser` that provides client credentials for mTLs]
 
-All keys are 2048 bits in size and are valid by default for 365 days from initial generation.
+When you apply the listener configuration for the Kafka cluster, a `<cluster_name>-cluster-ca-cert` secret is created with the certificates and keys to verify the cluster. 
```

Comment:
This happens when the KAfka cluster is deployed. Not when you add additional listeners.

Reply:
Updated